intakebuilder/getinfo.py

```diff
@@ -109,8 +90,7 @@ def getInfoFromGFDLDRS(dirpath,projectdir,dictInfo):
    #               "ensemble_member", "grid_label", "variable",
    #               "temporal subset", "version", "path"]
  
-#/archive/oar.gfdl.cmip6/ESM4/DECK/ESM4_historical_D1/gfdl.ncrc4-intel16-prod-openmp/pp/atmos/ts/monthly/5yr/DO_NOT_USE/atmos.201001-201412.alb_sfc.nc
-
+#Grab values based on their expected position in path 
     stemdir = dirpath.split("/")
     if stemdir[len(stemdir)-3] == "ts":
```

Comment:
nice that this works for our current use-case, but for time average or directory structure that the user defines slightly out of order than the usual way (as canopy allows this), the same logic will not quite scale up.

We should consider going back to builderconfig and seeing why it really broke things before, may be just a minor bug. builderconfig will allow users to indicate how their directory structure is. We can have the current updates merged in, but note that we have to revisit this and make it more robust and something that works for out of order patterns also, and something that can potentially work on time average with just a config update.